install_config/configuring_local.adoc

```diff
@@ -159,6 +159,45 @@ $ oc new-app -p CONFIGMAP=local-volume-config \
   -p SERVICE_ACCOUNT=local-storage-admin \
   -p NAMESPACE=local-storage local-storage-provisioner
 ----
+
+ifndef::openshift-origin[]
+<1> Replace `v3.9` with the right {product-title} version.
++
+endif::[]
+
+. Add the necessary storage classes:
++
+[source, bash]
+----
+oc create -f ./storage-class-ssd.yaml
+oc create -f ./storage-class-hdd.yaml
+----
++
+.storage-class-ssd.yaml
```

Comment:
I'm pretty against putting the yaml after the command to create the object that uses the yaml.